src/components/Title.tsx

```diff
@@ -1,5 +1,6 @@
 import style from "./Title.module.scss";
+import React from "react";
```

Comment:
Always put react first. It is not an error. But that is the usual practice, and it shows you care and you have checked that. That you want to leave the code behind in perfect shape, including these small details